pymc/model_graph.py

```diff
@@ -39,6 +40,47 @@
 )
 
 
+@dataclass
+class DimInfo:
+    names: tuple[str | None, ...]
+    lengths: tuple[int, ...]
+
+    def __post_init__(self) -> None:
```

Comment:
TIL

Reply:
nice in `pydantic.dataclasses.dataclass` as well